README.md

```diff
@@ -207,6 +209,18 @@ class Welcome_test extends TestCase
 
 See [How to Write Tests](https://github.com/kenjis/ci-phpunit-test/blob/master/docs/HowToWriteTests.md) for details.
 
+## (Optional) How to generate test result report
+
+~~~
+composer require phing/phing --dev
+~~~
+
+Use phpunit-frames.xsl file to replade file vendor/phing/phing/etc/phpunit-frames.xsl _//this change is used to show Assertions number in test report_  
+After run phpunit, it will generate a build folder.
+Make sure phpunit command could be run as global.  _/usr/local/bin/phpunit_  
+And then run this command "../vendor/phing/phing/bin/phing -f build.xml" after you run phpunit command in tests folder.  
```

Comment:
@biechao 
Fix the command path to work fine. And make it clear where you should be (application/tests/).

```
$ ../vendor/phing/phing/bin/phing -f build.xml
bash: ../vendor/phing/phing/bin/phing: No such file or directory
```

Reply:
@biechao Is this better?
```
vendor/bin/phing -f build.xml
```

Reply:
@biechao Dose not work. Please fix the instructions.

```
$ ../../vendor/bin/phing -f build.xml 
Buildfile: /home/kenji/work/codeigniter/ci-app-for-ci-phpunit-test/application/tests/build.xml
[autoloader] Loading autoloader from Bootstrap.php

BUILD FAILED
/home/kenji/work/codeigniter/ci-app-for-ci-phpunit-test/application/tests/build.xml:27:54: Provided autoloader file "./../vendor/autoload.php" is not a readable file

Total time: 0.0873 seconds
```

Reply:
@biechao I like I am always in the project root directory and run commands.
Why don't you change the current dirctory?

```
$ vendor/bin/phing -f build.xml
```